### DISCUSSION TITLE
请问旧版本g2中自定义 html 文本标签的 htmlTemplate 功能能否支持？目前有富文本标签需求。

### DISCUSSION THREAD

**Discussion by ChristmasFox** [antvis/G2#5322] [Q&A]
![image](https://github.com/antvis/G2/assets/68337739/db304595-e8ff-4c05-8e1d-1c7f9e2a478b)

**Comment by hustcc** [comment]
在开发中了：https://github.com/antvis/G2/issues/5133

**Reply by ChristmasFox** [reply]
好的
